package.json

```diff
@@ -58,19 +59,15 @@
     "yorkie": "^2.0.0"
   },
   "gitHooks": {
-    "pre-commit": "lint-staged",
+    "pre-commit": "lint-staged --concurrent false",
```

Comment:
This is needed, cause first we want to format and after the formatting we want to run eslint
So eslint will not pre-detect problems that are solved with prettier out of the box

Reply:
Nevermind... I found a better solution :smiley:

Reply:
> Nevermind... I found a better solution 😃

This breaks when you don't commit a .ts file but e.g. a single .css file?! So I reverted it to use concurrent=false again...